tests/JsonTests.tla

```diff
@@ -1,5 +1,15 @@
 ----------------------------- MODULE JsonTests -----------------------------
-EXTENDS Json, TLCExt
+EXTENDS Json, Naturals, TLCExt
+
+ASSUME(AssertEq(ToJson(<< >>), "[]"))
+ASSUME(AssertEq(ToJson(<<"a", "b", "c">>), "[\"a\",\"b\",\"c\"]"))
+ASSUME(AssertEq(ToJson([i \in 1..3 |-> "a"]), "{\"1\":\"a\",\"2\":\"a\",\"3\":\"a\"}"))
```

Comment:
Does it make sense to de-serialize a large/real-world .json file?

Reply:
Yeah, this is something left todo. I think the serialize/deserialize operators need tests.

Reply:
Maybe:

```tla
LET someTLA+Expr == ...
IN Deserialize(Serialize(someTLA+Expr, out.json), out.json) = someTLA+Expr
```

Reply:
I'll try it out